Fakes/OpenActive.FakeDatabase.NET.Tests/FakeBookingSystemTest.cs

```diff
@@ -56,7 +57,7 @@ public void Transaction_Effective()
 
             using (var db = FakeBookingSystem.Database.Mem.Database.Open())
             {
-                using (var transaction = db.OpenTransaction())
+                using (var transaction = db.OpenTransaction(IsolationLevel.Serializable))
```

Comment:
You could remove transaction.Dispose(); at line 64 since this using should do it for us.